composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": "^7.1",
-        "cocur/slugify": "^1.4 || ^2.0",
+        "cocur/slugify": "^3.0",
```

Comment:
Why not keep previous version contraints? Or at least keep "^2.0"? That would make the transition smoother.

Reply:
The idea is great, but that new languages rules are only available since version 3.0

Reply:
So what? with `"^1.4 || ^2.0 || ^3.0"`, you will be able to get them if you care, or to stick with v2 if you can't afford to upgrade because you use hhvm (I know, who does that, but it's an example)

Reply:
Amazing. I think yours concerns are valid.
Done 😄